Markup/samples/img/wso2-dark.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" width="14" height="14"><rect id="backgroundrect" width="100%" height="100%" x="0" y="0" fill="none" stroke="none"/>
```

Comment:
Document declaration is missing. Any reason ?
`<?xml version="1.0" encoding="utf-8"?>`

And the license header?

Reply:
Added document declaration and license header